app/Http/Controllers/Api/V1/OrganisationController.php

```diff
@@ -1393,4 +1393,86 @@ public function getRegistries(Request $request, int $id): JsonResponse
             throw new Exception($e->getMessage());
         }
     }
+
+    /**
+     * @OA\Patch(
+     *      path="/api/v1/organisations/{id}/approved",
+     *      summary="SuperAdmin update org system_approved flag",
+     *      description="Updates the system_approved flag for an organisation",
+     *      tags={"organisations"},
+     *      security={{"bearerAuth":{}}},
+     *      @OA\Parameter(
+     *         name="id",
+     *         in="path",
+     *         description="organisations entry ID",
+     *         required=true,
+     *         example="1",
+     *         @OA\Schema(
+     *            type="integer",
+     *            description="organisations entry ID",
+     *         ),
+     *      ),
+     *      @OA\RequestBody(
+     *          required=true,
+     *          description="Invite definition",
+     *          @OA\JsonContent(
+     *              @OA\Property(property="system_approved", type="bool", example="true"),
+     *          ),
+     *      ),
+     *      @OA\Response(
+     *          response=201,
+     *          description="Success",
+     *          @OA\JsonContent(
+     *              @OA\Property(property="message", type="string", example="success"),
+     *              @OA\Property(property="data", type="integer", example="1"),
+     *          ),
+     *      ),
+     *      @OA\Response(
+     *          response=500,
+     *          description="Error",
+     *          @OA\JsonContent(
+     *              @OA\Property(property="message", type="string", example="error")
+     *          )
+     *      )
+     * )
+     */
+    public function updateApproved(Request $request, int $id): JsonResponse
+    {
+        try {
+            $input = $request->only(app(Organisation::class)->getFillable());
+            $org = Organisation::findOrFail($id);
+
+            if (!Gate::allows('updateSro', Organisation::class)) {
```

Comment:
Sorry, Dan. Updated above function name, but the gate permission remains `updateSro`